test/test_video.py

```diff
@@ -244,11 +244,11 @@ def _template_read_video(video_object, s=0, e=None):
     video_frames = torch.empty(0)
     frames = []
     video_pts = []
-    for frame in itertools.takewhile(lambda x: x['pts'] <= e, video_object):
-        if frame['pts'] < s:
+    for frame in itertools.takewhile(lambda x: x["pts"] <= e, video_object):
```

Comment:
Why these `' -> "` changes?

Reply:
Probably black formatting, let's please avoid these changes until we can fully use black in torchvision (Philip and I are working on it https://github.com/pytorch/vision/issues/4174)

Reply:
A combination of black (or and my editor not picking up flake); sry, will revert